client/modules/PostEditor/PostEditor.tsx

```diff
@@ -119,6 +121,10 @@ function PostEditor(props: types.Props) {
     }
   );
 
+let name;
```

Comment:
You don't need those variables here - you can place them inside the JSX - below line 147

Reply:
or you can browse first category here `_.head(categories)` and spread them in the line 148